### ISSUE TITLE
bug: faulty file error while recording mocks

### ISSUE THREAD

**Issue by slayerjain** [keploy/go-sdk#158]
**Describe the bug**
In the recent versions of keploy (`v0.7.9+`), Keploy throws the error `failed to load yaml file` while recording mocks. 
 
**To Reproduce**
Steps to reproduce the behaviour:
1. Install keploy `v0.7.11` 
2. Install the mocking module
3. Trying recoding an HTTP mock
4. See the error

**Expected behaviour**
```
.2023-01-05T17:35:07.264+0100    INFO    server/server.go:214    keploy started at port 6789
2023-01-05T17:35:18.889+0100    ERROR    mock/mock.go:41    failed to load yaml file    {"with path": "/Users/sid.shukla/go/src/github.com/nutanix-cloud-native/prism-go-client/karbon/mocks/TestMetaOperations_GetVersion.yaml", "error": "stat /Users/sid.shukla/go/src/github.com/nutanix-cloud-native/prism-go-client/karbon/mocks/TestMetaOperations_GetVersion.yaml: no such file or directory"}
go.keploy.io/server/pkg/service/mock.(*Mock).FileExists
    /Users/sid.shukla/go/pkg/mod/go.keploy.io/server@v0.7.11/pkg/service/mock/mock.go:41
go.keploy.io/server/grpc/grpcserver.(*Server).StartMocking
    /Users/sid.shukla/go/pkg/mod/go.keploy.io/server@v0.7.11/grpc/grpcserver/server.go:60
go.keploy.io/server/grpc/regression._RegressionService_StartMocking_Handler
    /Users/sid.shukla/go/pkg/mod/go.keploy.io/server@v0.7.11/grpc/regression/services_grpc.pb.go:367
google.golang.org/grpc.(*Server).processUnaryRPC
    /Users/sid.shukla/go/pkg/mod/google.golang.org/grpc@v1.45.0/server.go:1282
google.golang.org/grpc.(*Server).handleStream
    /Users/sid.shukla/go/pkg/mod/google.golang.org/grpc@v1.45.0/server.go:1619
google.golang.org/grpc.(*Server).serveStreams.func1.2
    /Users/sid.shukla/go/pkg/mod/google.golang.org/grpc@v1.45.0/server.go:921
2023-01-05T17:35:20.124+0100    INFO    mock/mock.go:133
✅ Mocks are successfully written in yaml file at path: /Users/sid.shukla/go/src/github.com/nutanix-cloud-native/prism-go-client/karbon/mocks/TestMetaOperations_GetVersion.yaml
```
**Screenshots**
![PNG image](https://user-images.githubusercontent.com/12831254/210990467-a54710e5-ae20-4441-802e-89602123fba0.jpeg)

**Desktop (please complete the following information):**
 - OS: macOS

**Additional context**
reported by @thunderboltsid

**Comment by slayerjain** [comment]
@thunderboltsid this seems to be fixed automatically in keploy `v0.7.12`.  can you please check if its fixed for you too?

**Comment by thunderboltsid** [comment]
> @thunderboltsid this seems to be fixed automatically in keploy `v0.7.12`. can you please check if it's fixed for you too?

Yeah seems fine in `0.7.12`

**Comment by slayerjain** [comment]
Great. Closing the issue then
